app/views/spree/admin/payments/new.html.haml

```diff
@@ -1,4 +1,4 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Payments' }
```

Comment:
same here, needs a call to title and order_tabs

Reply:
one extra point about notation, you should always avoide hash rockets => wherever possible.
Should be partial: not :partial =>

Reply:
added order_tabs back in and removed rocket